go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 	github.com/secure-systems-lab/go-securesystemslib v0.8.0
 	github.com/segmentio/ksuid v1.0.4
+	github.com/sigstore/protobuf-specs v0.2.2-0.20240118022641-cea4265fd6c5
```

Comment:
Can this be v0.3.0rc0? Or is that not a valid version in golang?

Reply:
When I add that tag to `go.mod`, `go mod tidy` corrects it to this version. Judging from [this](https://go.dev/doc/modules/version-numbers), it probably should be v0.3.0-rc0 rather than v0.3.0rc0.

Reply:
Hmm, this is something I can probably unilaterally fix -- I can push up a `v0.3.0-rc1` tag to protobuf-specs. Once that's in, you should be able to bump here.

Reply:
@tetsuo-cpp I've created `v0.3.0-beta.2`: https://github.com/sigstore/protobuf-specs/releases/tag/v0.3.0-beta.2

Reply:
Awesome, I'll use that tag.

Reply:
[c14bfce](https://github.com/sigstore/sigstore/pull/1601/commits/c14bfce5f9b2b5018574faae4b8fa5cb7474608f)